boost/plan.sh

```diff
@@ -25,15 +27,21 @@ pkg_build_deps=(
   core/libxslt
   core/openssl
   core/which
+  core/zlib
 )
 
 pkg_lib_dirs=(lib)
 pkg_include_dirs=(include)
 
 do_build() {
-  ./bootstrap.sh --prefix=$pkg_prefix
+  ./bootstrap.sh --prefix="$pkg_prefix"
 }
 
 do_install() {
-  ./b2 install --prefix=$pkg_prefix -q --debug-configuration -s NO_BZIP2=1
+  export NO_BZIP2=1
+  export ZLIB_LIBPATH
+  ZLIB_LIBPATH="$(pkg_path_for core/zlib)/lib"
+  export ZLIB_INCLUDE
+  ZLIB_INCLUDE="$(pkg_path_for core/zlib)/include"
```

Comment:
These declare+assignments were broken up per shellcheck's [SC2155](https://github.com/koalaman/shellcheck/wiki/SC2155) "Declare and assign separately to avoid masking return values." I notice though that there are several core plans that don't follow this pattern and do the declare+assignment together.